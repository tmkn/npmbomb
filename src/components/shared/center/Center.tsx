/** @jsx jsx */
import { jsx, css } from "@emotion/core";

const centerStyle = css({
    display: "flex",
    flex: 1,
    justifyContent: "center"
});

export const Center: React.FC = ({ children }) => {
    return <div css={centerStyle}>{children}</div>;
};
