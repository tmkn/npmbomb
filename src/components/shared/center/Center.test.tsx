import React from "react";
import { render, fireEvent, waitForElement } from "@testing-library/react";

import { Center } from "./Center";

describe("<Center />", () => {
    test("matches snapshot", () => {
        const { asFragment } = render(<Center>Test</Center>);

        expect(asFragment()).toMatchSnapshot();
    });
});
