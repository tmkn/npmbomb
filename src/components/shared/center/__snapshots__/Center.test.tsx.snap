// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Center /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-8a5j6a"
  >
    Test
  </div>
</DocumentFragment>
`;
